### DISCUSSION TITLE
Welcome to Hacktoberfest Discussions!

### DISCUSSION THREAD

**Discussion by umershaikh123** [umershaikh123/Hacktoberfest#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Shaba1406** [comment]
can you please tell me why it is showing excluded to me ?

![Color Picker - Google Chrome 04-10-2022 00_09_18](https://user-images.githubusercontent.com/97939567/193653789-835f8e1a-27ee-4241-80ee-8ed3b5d56cdc.png)

**Comment by umershaikh123** [comment]
I am sorry everyone but it seems the rules for hacktoberfest 2022 are different than before.
It was unclear and confusing but after talking in their discord I found out that the contributions needs to be made to an existing projects ,
issues and must be solving something and not just any piece of code.

So you might want to contribute to a different repository than this one.

Sorry for the inconvenience

**Comment by Shaba1406** [comment]
Okay no worries, its fine 👍: )
